user/deployment/npm.md

```diff
@@ -132,3 +132,36 @@ after_deploy:
   - ./after_deploy_1.sh
   - ./after_deploy_2.sh
 ```
+
+### Troubleshooting common problems
+
+#### Skipping a deployment with the npm provider because this is not a tagged commit
+
+The most likely cause is that `git push` does not push tags. Be sure to run `git push --tags`.
+
+#### Skipping a deployment with the npm provider because this is not on the required runtime
+
+This is because the node version being used for the test does not match the version specified in the conditonal release section:
```

Comment:
Hmm, you're right... how about "This happens when..."? I like the sound of that.